From 082598d7a4be0038ed87ce3af891ae1e3b23d9c3 Mon Sep 17 00:00:00 2001
From: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
Date: Fri, 31 Jul 2020 15:08:16 +0700
Subject: [PATCH 427/441] arm64: dts: renesas: r8a774a3-hihope-rzg2m: Add
 display clock properties

Add display clock properties for the HiHope RZ/G2M v3.0 board.

Signed-off-by: Khoi Nguyen <khoi.nguyen.aj@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
index 25787ec..2beb827 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a3-hihope-rzg2m.dts
@@ -90,3 +90,14 @@
 &ohci1 {
 	memory-region = <&global_cma>;
 };
+
+&du {
+	clocks = <&cpg CPG_MOD 724>,
+		<&cpg CPG_MOD 723>,
+		<&cpg CPG_MOD 722>,
+		<&versaclock5 1>,
+		<&x302_clk>,
+		<&versaclock5 2>;
+	clock-names = "du.0", "du.1", "du.2",
+		"dclkin.0", "dclkin.1", "dclkin.2";
+};
-- 
2.7.4

